atlasdb-api/src/main/java/com/palantir/atlasdb/encoding/PtBytes.java

```diff
@@ -67,56 +68,56 @@ public static long toLong(byte[] bytes, int offset) {
     /**
      * Converts a string to a UTF-8 byte array.
      */
-    public static byte[] toBytes(String s) {
-        return s.getBytes(Charsets.UTF_8);
+    public static byte[] toBytes(String str) {
+        return str.getBytes(StandardCharsets.UTF_8);
     }
 
     /**
      * Converts a string to a UTF-8 byte array. This method will cache results and return cached
      * results, so the resulting byte arrays must not be modified.
      */
-    public static byte[] toCachedBytes(String s) {
-        byte[] bytes = cache.getIfPresent(s);
+    public static byte[] toCachedBytes(String str) {
```

Comment:
But `arr` is just as bad as `a` - I think we might be strongly straying into the realm of blindly obeying rules rather than having static analysis prompting us to reconsider a choice.